.travis.yml

```diff
@@ -1,8 +1,25 @@
 services:
-    - docker
+- docker
 
-script: docker-compose run test
+script: docker-compose run windbreaker npm run test:docker
+
+after_success:
+  # windbreaker/windbreaker:latest is automatically built and tagged
+  # from test run
+  |
+    if [ $TRAVIS_BRANCH == "master" ]; then
+      docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
+      docker tag windbreaker/windbreaker:latest windbreaker/windbreaker:build-$TRAVIS_BUILD_NUMBER;
+
+      # push both the build tag and update latest
+      docker push windbreaker/windbreaker:$TRAVIS_BUILD_NUMBER;
+      docker push windbreaker/windbreaker:latest;
+    fi
 
 notifications:
   slack:
     secure: J97h63QLRfK2DV41dYv42T0b6mRBxe3joDQP2EeEjpyIYQ52i2so6m5NA1GVJydzQ1uEYcxicX6GVeox0lt3PSqPAMYr9Tb/7pJ9mBOAAEoSpkgRcNsMItbEkJGGL5IfvTEy9/2JiXhrNbrCGlJepO72zq/Hjkc5oIYs7PTNCAhXk3Ys0HyQKnHxvwq3DAk2t39PaLAoNDbZuqZ95SqbvZJjehjv6fXolUkuwT090rThyCK43ATaDaXKWFEIVsiYxnsiu7TLqvIG1b2mLa2Zo0giQrH+5DOrT8a73ZJHdPzxI1OknC0Ezy5GdUsXaIV4Bqhdb2CMGexY+XieBn/hMgWFD+yic5vVvnXh0Z3QBIROR4T2PF6ubZfick9Yx7/z+eo/HEIZW7QpKN+4y0hBaK2KQr6WyXCMg8xoTEFgdakqgYtZdc/egTlHb1KrSYxviNkgidVIiApbTzhnVRwB/Pj727m9g2ffn73nS7GDzMUxUvIE6L8tk238JEmx77XmkFAKjdAeTAf/eCm9J6nZH8kJ8I1EMS08gUymacBb5fbUb8mRSmNPnTKCgMjwkKPaKMDyqqZcdu7sYv/lfvH4/jtuc0pXSIeOoi3hoNFmbazf3xnigS4Vn0rDw1UpZ65ZLSCCP1mtbfN1fXekhtuOfxeJXwn49SZnyt6vwOhiQDA=
+env:
+  global:
+  - secure: Hj3AndwXq9RyXOrNPOsZk4v5stxVeqOegM7BGKJAlRPDkF5X4MxWXX6b1ZbGrBgmz76UwNHmFkZxcYW2loE2M873mRUaIIQruR7XI9bTg5UbF5W1F+JtUpPRTLAJpA4j2pcTyWpdnlQSEBJT2Y1xq0lkXEANfRuiribnx3GZiDEqXngQZ49Z51/l9OPvvDhGAAkb1bEkL8Gavgkm0grRVAL+QsaDPzzbCOCa11PgNMGkxPPDtetAiBUiLX441pnhqY0OfQAK6nBMySm4/HQBUbWCdyl9//zE8AZT8Xmek2jNhLmFakS3ffq0XRD+VWeeHjnZu1TytDJvZhWBe5rs+WWsyne66sQ90ALVOsbGzOHcVK3Lj95EK4nXqYdgd0jtte2rQStM2BJoEQDHaDZYoRD4cWWgQAIGDIpu2iTxVMCXjn8KTD3NwDd7DEcgXeQM8bni0WQ1A7Do/hwNYBtBa7iqlE4Yhob5ijs24xw6C99ztB9DXyylZA9ltXAaJxmBeXFcdPf8xL2KZf+rchDYZUHZVCY6pSNoaxnum3CabWdNfWd21aCER3hwGA3HF6WlxQTgyfhtXoF+MsuKYDJ50LBnhIqh6YcNWKPK1vjgWsS9yS/omYcas9TOGLUVwsumLwAc4gbDZ1/nh8Egyrz+2tVAvtZ/bw2gPfAbxhopgAM=
+  - secure: bjgEBxpEp4sY896h9hruzszBlpPs78PjDA2/Mh9pzHAOGAWHsHD8R/uzY7rAUL9yNK8i5rTgNV/KTMLL7PQwczKCOMGPDBY4OCJuxwarTy6JEaRquyLh+ZacGtwu76J8KFY+h1qkqsTJtRmd8avNfuLku9lUdhDv7l6LHzlYbT7rHRdSyj3QIopb9snvJTckxK6kitibsadWYx2srKKtoiLA2XX6xBLZ/KeYcRCEKHdHOyTOPlwwlSJcSRHD6F6di9Ufa5XTOoM4mt94tyBJXuMc6ccgRUIXiggEiterAmJD/B3RJwNobQREq1JcLZdyP5zjMY13udRINwbV06I8amnqDSHSNZvvXQylY3eO2XMV5/i5j6Furc3qHrO9ENd6kw0QDmxHiVmzVh7AnCb6wKkcrOdi4rOrw7Y5/7Ag7eT6L2zj3KEBtHjcQgWK905c/6yoCD6olGI5orsG3dzIEEvALBPF4C34krBY6rshJYiUJfCn7Q0asvqj63Z1mYGON0EgQxJ2H+b06QJZOE8xeEFthb/pGFUOuKamSk5VglIpk31sO+IVs10WUW6bHE7tM6KpqC1VCilXPaTVohqndzlwFazhwPPQP/9lvPQwJEwOlgzR3cSenmV2OCh+gxxYhjetAv6+obXfC85tO+X/Gu7FMN6GweK8zyopteh8Ucw=
```

Comment:
I created a dedicated windbreaker team email so that we can start tying all windbreaker related accounts to it (dockerhub, aws, etc.).